homeassistant/components/zcs_azzurro_inverter/__init__.py

```diff
@@ -0,0 +1,49 @@
+"""The zcs_azzurro integration."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up zcs_azzurro from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
+
+    await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
+
+    return True
+
+
+async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Unload a config entry."""
+    if unload_ok := await hass.config_entries.async_unload_platforms(entry, PLATFORMS):
+        hass.data[DOMAIN].pop(entry.entry_id)
+
+    return unload_ok
+
+
+class CannotConnect(HomeAssistantError):
+    """Error to indicate we cannot connect."""
+
+
+class InvalidAuth(HomeAssistantError):
+    """Error to indicate there is invalid auth."""
+
+
+class ResponseError(HomeAssistantError):
+    """Error to indicate there was a not ok response."""
+
+
+class ConfigEntryNotReady(HomeAssistantError):
+    """Error to indicate the entry is unreachable."""
+
+
+class ConfigEntryAuthFailed(HomeAssistantError):
+    """Error to indicate the auth to the entry failed."""
```

Comment:
You shouldn't be creating them, you should be importing them
from homeassistant.exceptions import ConfigEntryAuthFailed, ConfigEntryNotReady